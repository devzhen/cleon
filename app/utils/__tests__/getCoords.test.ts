import getCoords from '../getCoords';

describe('getCoords', () => {
  it('return coords', () => {
    const elem = document.createElement('div');
    elem.getBoundingClientRect = jest.fn(
      () =>
        ({
          x: 10,
          y: 10,
          width: 100,
          height: 100,
          top: 10,
          left: 10,
        }) as DOMRect,
    );

    const coords = getCoords(elem);

    expect(coords).toMatchSnapshot();
  });
});
