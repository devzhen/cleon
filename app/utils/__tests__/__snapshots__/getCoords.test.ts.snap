// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getCoords return coords 1`] = `
{
  "bottom": 110,
  "height": 100,
  "left": 10,
  "right": 110,
  "top": 10,
  "width": 100,
}
`;
